test/functional/lua/vim_spec.lua

```diff
@@ -327,4 +327,61 @@ describe('lua stdlib', function()
     -- error handling
     eq({false, 'Vim:E714: List required'}, exec_lua([[return {pcall(vim.fn.add, "aa", "bb")}]]))
   end)
+
+  it('vim.validate', function()
+    eq(true, pcall(exec_lua, "return vim.validate({ arg1={ {}, 'table' }})"))
```

Comment:
There's no reason to use `pcall` if you expect success. `pcall` is only for testing an expected failure.